### ISSUE TITLE
image_retraining/retrain.py warning: Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization()

### ISSUE THREAD

**Issue by MicrocontrollersAndMore** [tensorflow/tensorflow#16927]
If I download https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/image_retraining/retrain.py and run it I get the following warning:

`2018-02-10 20:48:38.928435: W C:\tf_jenkins\workspace\rel-win\M\windows\PY\36\tensorflow\core\framework\op_def_util.cc:343] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().`

Here is a screenshot if that helps:

![untitled](https://user-images.githubusercontent.com/5672876/36068513-456022f4-0e8c-11e8-8b81-d0d5fb5199b0.png)

After looking at some old TensorFlow issues, for example these:

https://github.com/tensorflow/tensorflow/issues/2164

https://github.com/tensorflow/tensorflow/issues/4128

It seems that this concern has existed for quite a while and that it seems likely related to using the Inception model from 2015:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/image_retraining/retrain.py#L893

Can anybody update this to resolve this warning please?

-- Edit ------------------------

provided additional requested information in separate response below

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by MicrocontrollersAndMore** [comment]
Have I written custom code:
No custom code, I ran retrain.py with no modifications

OS Platform and Distribution:
Windows 10

TensorFlow installed from:
pip

TensorFlow version:
1.5

Bazel version:
not sure

CUDA/cuDNN version:
n/a (using CPU version of TensorFlow)

GPU model and memory:
n/a (using CPU version of TensorFlow)

Exact command to reproduce:
shown in screenshot above

**Comment by bignamehyp** [comment]
Thanks for your input. We will migrate to tf.nn.batch_normalization shortly.

**Comment by gkorland** [comment]
@bignamehyp Was the example ported to use  tf.nn.batch_normalization? I'm still getting this warning with the latest.
